### ISSUE TITLE
References for Measles-Experiment

### ISSUE THREAD

**Issue by caijun** [mattg3004/Measles-Experiment#2]
Hi @mattg3004 , I am interested in your Measles-Experiment code. To facilitate my understanding, could you provide some references about the code, such as papers. Thank you very much.

**Comment by mattg3004** [comment]
Hello,
The code is based on the model from this paper: http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4086157/ which I’ve also attached. To be honest, I’ve not documented the code very well, so it is probably just as simple to recreate the model for yourself using the description from the paper.
Best,
Matt

On March 17, 2016 at 12:23:05 PM, Tony Tsai (notifications@github.com) wrote:

Hi @mattg3004 , I am interested in your Measles-Experiment code. To facilitate my understanding, could you provide some references about the code, such as papers. Thank you very much.

**Comment by caijun** [comment]
Thank you for your quick reply.

**Comment by Spatial-R** [comment]
After running the codes after **3000*time.step**, i have come across  some outbreaks, such as 1000 case per day during the period. However, in reality, such outbreaks will not happen, owing to the artificial interventions to the outbreak or the report probability. How to judge the validity of simulation?

**Comment by mattg3004** [comment]
Yes, you’re right. The simulation only simulates infections, so if you want to work out the amount of reported cases, then you have to multiply these by the proportion that are reported. It also doesn’t add artificial interventions during the outbreaks, so if you want to include these, then you must add them to the code, be that by removing people through vaccination, or some other method. 

On March 20, 2016 at 11:15:33 PM, Bing Zhang (notifications@github.com) wrote:

After running the codes after 3000*time.step, i have come across some outbreaks, such as 1000 case per day during the period. However, in reality, such outbreaks will not happen, owing to the artificial interventions to the outbreak or the report probability. How to judge the validity of simulation?
